### ISSUE TITLE
Glide width parameters not setting max width

### ISSUE THREAD

**Issue by joetannenbaum** [spatie/statamic-responsive-images#26]
I can't tell if I'm not understanding the purpose of the `glide:width` parameters or if there's a bug, but here's the example situation:

I have an image that's 100px wide, I want to set the max-width to 50px. If I do this:

`{{ responsive:image glide:width="50" }}`

nothing changes about the output. I dove into the source code and I found that if I changed the line here:

https://github.com/spatie/statamic-responsive-images/blob/master/src/Responsive.php#L172

to this instead:

```php
/* We want at least one width to be returned */
if (!$filtered->count()) {
    $filtered = collect($params['width'] ?? $params['w']);
}
```

all works as intended. Am I misunderstanding or misusing this attribute? Or is the above a fix for this? Happy to submit a PR if so.

**Comment by riasvdv** [comment]
It seems in your case the WidthCalculator isn't returning any sizes < 50, could you verify this by doing a dump of the `$widths` variable here https://github.com/spatie/statamic-responsive-images/blob/master/src/Responsive.php#L162

**Comment by joetannenbaum** [comment]
This is what I get (the real width of the original image was 429px, I just used 100px to make the question a bit cleaner):

![image](https://user-images.githubusercontent.com/2702148/102798973-b0fc9300-4366-11eb-8538-ae561a761060.png)

**Comment by riasvdv** [comment]
Ok, that explains. I think your solution is a good one, if there's no valid width we'll just take the max width that was specified. Happy to merge a PR with your change from above.

**Comment by riasvdv** [comment]
Fixed in 1.7.1!
